test/testdrive/oom.td

```diff
@@ -12,104 +12,67 @@
 $ postgres-connect name=mz_system url=postgres://mz_system:materialize@${testdrive.materialize-internal-sql-addr}
 
 $ postgres-execute connection=mz_system
-ALTER SYSTEM SET max_result_size = 128
+ALTER SYSTEM SET max_result_size = '1MB'
 
 # In environmentd each Row requires 24 bytes + number of bytes to encode. But we'll also de-dupe
 # repeated values by incrementing the diff.
 
-> SELECT 1::int4 FROM generate_series(1, 10);
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-
-! SELECT * FROM generate_series(1, 6);
-contains:result exceeds max size of 128 B
-
-> CREATE TABLE t1 (a int4)
+> SELECT 1::int4 FROM generate_series(1, 10000), repeat('a', 100);
+10000 values hashing to b223cca8b360eae4e49568512e2de29f
```

Comment:
TIL this works, pretty neat.

Reply:
It's used somewhere below in the same file, which I how I learned about it :D